### PR TITLE
Extend DZE_BackpackAntiTheft to support groups

### PR DESCRIPTION
**Pull request by oiad**
This change allows group members as well as players you are friendly to
to access each others gear.

This also fixes the problem with being able to access peoples gear in
safe zones by just tagging them as a friend, this was quite dangerous
because it required no authorization from the remote player. Now both
players need to be friendly to each other OR in a group (group members
can be considered trust worthy, I guess?)

Fixes an issue in fn_check_access where it was trying to get ownerPUID
on a player instead of getPlayerUID

### CODE REVIEW

File: SQF/dayz_code/stringtable.xml [hunk 1]
```diff
@@ -13092,7 +13092,7 @@
 			<Czech>%1 je omezeno.</Czech>
 		</Key>
 		<Key ID="STR_EPOCH_PLAYER_316">
-			<English>You may only access friends' gear in this area.</English>
+			<English>You may only access friends or group members gear in this area.</English>
 			<German>Du kannst hier nur auf das Inventar von Freunden zugreifen.</German>
```
Comment (AirwavesMan) [inline SQF/dayz_code/stringtable.xml]: Can you  update the german string with that?

<German>Du kannst hier nur auf das Inventar von Freunden oder Gruppenmitgliedern zugreifen.</German>

File: SQF/dayz_code/compile/fn_gearMenuChecks.sqf [hunk 2]
```diff
@@ -29,9 +29,13 @@ if ((vehicle player) == player) then {
...
 		};
 		if (DZE_BackpackAntiTheft) then {
-			_friendlies	= player getVariable ["friendlies",[]];
+			_friendlyTo = player getvariable ["friendlyTo",[]];
 			_rID = if (DZE_permanentPlot) then { getPlayerUID _cTarget } else { _cTarget getVariable ["CharacterID","0"] };
-			if ((!canbuild or isInTraderCity) && {alive _cTarget} && {isPlayer _cTarget} && {!(_rID in _friendlies)} && {(player distance _cTarget) < 12}) then {
+			_groupies = [];
+			{
+				_groupies set [count _groupies,getPlayerUID _x];
+			} count (units (group player));
```
Comment (ebayShopper) [inline SQF/dayz_code/compile/fn_gearMenuChecks.sqf]: No need to get UIDs of all group members, you can just do:
```sqf
!(_cTarget in (units group player))
```

File: SQF/dayz_code/compile/fn_gearMenuChecks.sqf [hunk 3]
```diff
@@ -1,4 +1,4 @@
-private ["_exit","_nearestObjects","_friendlies","_rID","_display","_cTarget","_dis"];
+private ["_exit","_nearestObjects","_friendlies","_rID","_display","_cTarget","_dis","_groupies","_friendlyTo"];
```
Comment (ebayShopper) [inline SQF/dayz_code/compile/fn_gearMenuChecks.sqf]: You can remove "_friendlies" from this now.

File: SQF/dayz_code/stringtable.xml (hunk 1 above)
Comment (oiad) [inline SQF/dayz_code/stringtable.xml]: Done

**Comment by oiad** [comment]
@ebayShopper Changes all made

**Pull request closed without merging**